frontend/src/Settings/AdvancedSettingsButton.js

```diff
@@ -42,18 +43,27 @@ function AdvancedSettingsButton(props) {
         />
       </span>
 
-      <div className={styles.labelContainer}>
-        <div className={styles.label}>
-          {advancedSettings ? 'Hide Advanced' : 'Show Advanced'}
-        </div>
-      </div>
+      {
+        showLabel ?
+          <div className={styles.labelContainer}>
+            <div className={styles.label}>
+              {advancedSettings ? 'Hide Advanced' : 'Show Advanced'}
```

Comment:
```suggestion
              {advancedSettings ? translate('Hide Advanced') : translate('Show Advanced')}
```

Reply:
I think this was completely removed based on this comment: https://github.com/Sonarr/Sonarr/pull/5646#pullrequestreview-1425724887

We had some further discussion in https://github.com/Sonarr/Sonarr/pull/5640 which makes a lot of sense for consistency across the different *arr apps.